plugin/zoom/lib/zoom_plugin.class.php

```diff
@@ -0,0 +1,466 @@
+<?php
+
+/* For licensing terms, see /license.txt */
+
+use Chamilo\PluginBundle\Zoom\API\CreatedRegistration;
+use Chamilo\PluginBundle\Zoom\API\JWTClient;
+use Chamilo\PluginBundle\Zoom\API\MeetingInstance;
+use Chamilo\PluginBundle\Zoom\API\MeetingSettings;
+use Chamilo\PluginBundle\Zoom\API\ParticipantListItem;
+use Chamilo\PluginBundle\Zoom\API\PastMeeting;
+use Chamilo\PluginBundle\Zoom\API\RecordingMeeting;
+use Chamilo\PluginBundle\Zoom\CourseMeeting;
+use Chamilo\PluginBundle\Zoom\CourseMeetingInfoGet;
+use Chamilo\PluginBundle\Zoom\CourseMeetingListItem;
+use Chamilo\PluginBundle\Zoom\UserMeetingRegistrant;
+use Chamilo\PluginBundle\Zoom\UserMeetingRegistrantListItem;
+
+class ZoomPlugin extends Plugin
```

Comment:
Codacy found an issue: [The class ZoomPlugin has 16 non-getter- and setter-methods. Consider refactoring ZoomPlugin to keep number of methods under 10.](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5625225)